cli/src/semgrep/cli.py

```diff
@@ -12,48 +12,15 @@
 from semgrep.commands.scan import scan
 from semgrep.default_group import DefaultGroup
 from semgrep.state import get_state
-from semgrep.util import git_check_output
 from semgrep.verbose_logging import getLogger
 
 logger = getLogger(__name__)
 
 
-def maybe_set_git_safe_directories() -> None:
-    """
-    Configure Git to be willing to run in any directory when we're in Docker.
-
-    In docker, every path is trusted:
-    - the user explicitly mounts their trusted code directory
-    - r2c provides every other path
-
-    More info:
-    - https://github.blog/2022-04-12-git-security-vulnerability-announced/
-    - https://github.com/actions/checkout/issues/766
-    """
-    env = get_state().env
-    if not env.in_docker:
-        return
-
-    try:
-        # "*" is used over Path.cwd() in case the user targets an absolute path instead of setting --workdir
-        git_check_output(["git", "config", "--global", "--add", "safe.directory", "*"])
-    except Exception as e:
-        logger.info(
-            f"Semgrep failed to set the safe.directory Git config option. Git commands might fail: {e}"
-        )
-
-
 @click.group(cls=DefaultGroup, default_command="scan", name="semgrep")
 @click.help_option("--help", "-h")
 @click.pass_context
 def cli(ctx: click.Context) -> None:
-    """
-    To get started quickly, run `semgrep scan --config auto`
-
-    Run `semgrep SUBCOMMAND --help` for more information on each subcommand
-
-    If no subcommand is passed, will run `scan` subcommand by default
-    """
```

Comment:
Why delete this documentation? It's user facing (prints when running `semgrep --help`), so if it's out of date I think we should instead update it

Reply:
Because this is now provided by osemgrep. cli/bin/semgrep now calls by default osemgrep which does handle the 'semgrep --help'.

Reply:
Ahh ok I see